# `scipy.sparse.SparseWarning`

> 原文：[`docs.scipy.org/doc/scipy-1.12.0/reference/generated/scipy.sparse.SparseWarning.html#scipy.sparse.SparseWarning`](https://docs.scipy.org/doc/scipy-1.12.0/reference/generated/scipy.sparse.SparseWarning.html#scipy.sparse.SparseWarning)

```py
exception scipy.sparse.SparseWarning
```

```py
with_traceback()
```

Exception.with_traceback(tb) – 设置 self.__traceback__ 为 tb 并返回 self。
